src/components/data/questionsData.json

```diff
@@ -1,94 +1,50 @@
+  },
+  {
+    "id": 2,
+    "question": {
+      "text": "What do you choose?",
       "image": "q2_trees_2.png",
       "options": [{
-        "choiceid": 3,
-        "option": "Choose and move to 4",
-        "nextId": 2
-      },
-      {
-        "choiceid": 4,
-        "option": "Choose and move to 5",
-        "nextId": 3
-      }
-    ]
+          "choiceId": 1,
+          "option": "Choose and move to 2",
+          "densityEffect": -1,
+          "infrastructureEffect": -1,
+          "nextId": null
+        },
+        {
+          "choiceId": 2,
+          "option": "Choose and move to 3",
+          "densityEffect": -1,
+          "infrastructureEffect": -1,
+          "nextId": null
+        }
+      ]
     },
-    {
-      "id": 3,
-      "question": "What do you choose?",
-      "image": "q3_earthquake_1.png",
-      "options": [{
-        "choiceid": 5,
-        "option": "Choose and move to 2",
-        "nextId": 2
-      },
-      {
-        "choiceid": 6,
-        "option": "Choose and move to 3",
-        "nextId": 3
-      }
-    ]
-    },
-    {
-      "id": 4,
-      "question": "What do you choose?",
-      "image": "q4_bicycle_1.png",
-      "options": [{
-        "option": "Choose and move to 2",
-        "nextId": 2
-      },
-      {
-        "option": "Choose and move to 3",
-        "nextId": 3
-      }
-    ]
-    },
-    {
-      "id": 5,
-      "question": "What do you choose?",
-      "image": "q5_people_1.png",
-      "options": [{
-        "option": "Choose and move to 2",
-        "nextId": 2
-      },
-      {
-        "option": "Choose and move to 3",
-        "nextId": 3
-      }
-    ]
-    },
-    {
-      "id": 6,
-      "question": "What do you choose?",
-      "image": "q6_law_1.png",
-      "options": [{
-        "choiceid": 5,
-        "option": "Choose and move to 2",
-        "nextId": 2
-      },
-      {
-        "choiceid": 6,
-        "option": "Choose and move to 3",
-        "nextId": 3
-      }
-    ]
-    }
-  ]
\ No newline at end of file
+    "event": "Test",
+    "fact": "Testing"
+  }
+]
```

Comment:
fake data for testing

Reply:
LGTM, I think there is also TransportationEffect right?

Reply:
I think that's a typo and should be infrastructure 😆  Kelsey said there are only 2 variables